packages/atomic/src/components/result-lists/result-list-common.tsx

```diff
@@ -235,38 +237,46 @@ export class ResultListCommon {
       return;
     }
 
+    if (resultListState.firstSearchExecuted && !resultListState.hasResults) {
+      return;
+    }
+
+    const displayPlaceholders = !isAppLoaded(this.bindings.store);
+
     const classes = this.getClasses(
       display,
       density,
       imageSize!,
       resultListState.firstSearchExecuted,
-      resultListState.isLoading
+      resultListState.isLoading,
+      displayPlaceholders
     );
     return (
       <Host>
         {templateHasError && <slot></slot>}
-        <div
-          class={`list-wrapper placeholder ${classes}`}
-          ref={setListWrapperRef}
-        >
+        <div class={`list-wrapper ${classes}`} ref={setListWrapperRef}>
           <ResultDisplayWrapper classes={classes} display={display}>
-            <ResultsPlaceholder
-              display={display}
-              density={density}
-              imageSize={imageSize}
-              resultsPerPageState={resultsPerPageState}
-            />
-            <Results
-              classes={classes}
-              bindings={this.bindings}
-              host={host}
-              display={display}
-              density={density}
-              imageSize={imageSize}
-              resultListState={resultListState}
-              resultListCommon={this}
-              getContentOfResultTemplate={getContentOfResultTemplate}
-            />
+            {displayPlaceholders && (
+              <ResultsPlaceholder
+                display={display}
+                density={density}
+                imageSize={imageSize}
+                resultsPerPageState={resultsPerPageState}
+              />
+            )}
+            {resultListState.firstSearchExecuted && (
```

Comment:
should this be 
```js
{displayPlaceholders && <Results...
```
so we guarantee that it's always one or the other?

Reply:
Results & placeholders will coexist in the dom for a bit until the results are truly loaded